### PR TITLE
Cards front responsive OK

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by RomainLF — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/Cards/Style.jsx [hunk 1]
```diff
@@ -1,164 +1,138 @@
...
-      margin-left: 0.1rem;
-      background-color: #ffffff7f;
-    }
-    .speed {
-      border: 0.1mm #ffffff7f;
-      width: 0.7rem;
-      margin-left: 0.1rem;
-      background-color: rgba(255, 255, 255, 0.5);
-    }
+  .namePower ul {
```
Comment (LoicBrassart) [inline frontend/src/components/Cards/Style.jsx]: Choose one (namePower) and stick with it

File: frontend/src/components/Jarvis/index.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { useState, useEffect } from "react";
 import axios from "axios";
 import SJarvis from "./style";
 import Modal from "./Modal";
-import Cards from "../Cards";
+import Card from "../Cards";
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/index.jsx]: That's a terrible idea: you should rename the file to match it's component name instead

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils, plz fix !

**Review by RomainLF — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
No more Evils!

**Pull request merged**